ij-server/src/main/kotlin/org/jetbrains/research/ij/headless/server/SingleFileProjectConfigurator.kt

```diff
@@ -40,4 +36,13 @@ object SingleFileProjectConfigurator {
         logger.info("New file for language has been successfully created")
         return SingleFileProject(language, project, file.get(), disposable)
     }
+
+    enum class MainFile(val languageId: String, val fileName: String) {
+        KotlinMainFile(LanguageId.kotlin.name, "Main.kt"),
```

Comment:
Let's rename into KotlinFile, you don't need to indicate that this file is the main one since you already inside the MainFile class